content/applications/sales/crm/acquire_leads/lead_mining.rst

```diff
@@ -20,61 +23,84 @@ Generate leads
+- :guilabel:`States`: further filter leads based on the state in which they are located, if
+  applicable.
+- :guilabel:`Industries`: filter leads based on the specific industry they work in.
+- :guilabel:`Filter on Size`: tick this checkbox to specify the number of employees at the company.
+  This generates a field labeled :guilabel:`Size`. Fill in the blanks to create a range for the
+  desired company size.
+- :guilabel:`Sales Team`: choose which Sales Team the leads will be assigned to.
+- :guilabel:`Salesperson`: choose which member of the Sales Team the leads will be assigned to.
+- :guilabel:`Default Tags`: choose which tags are applied directly to the leads once found.
 
 .. note::
    Make sure to be aware of the latest EU regulations when receiving contact information. Get more
    information about the General Data Protection Regulation on `Odoo GDPR <http://odoo.com/gdpr>`_.
 
-Pricing
-=======
+View leads
+----------
 
-Lead mining is an *In-App Purchase* feature and each generated lead costs one credit.
+After leads are generated, they are assigned to the designated salesperson and team. To view
+additional information regarding the lead, select one from the list and click to open it.
 
-Choosing to generate :guilabel:`Companies and their Contacts` costs one additional credit for each
-contact generated.
+In the *chatter* thread for the lead, additional information is provided. This can include the
+number of employees, the technology used by the company, the timezone, and direct contact
+information.
+
+.. image:: lead_mining/generated-lead.png
+   :align: center
+   :alt: The chatter thread of a newly generated lead.
 
 .. note::
-   See here for full pricing information: `Lead Generation by Odoo IAP <https://iap.odoo.com/iap/
-   in-app-services/167?>`_.
+   If :guilabel:`Leads` are **not** enabled for the database, then leads are generated as
+   *opportunities* and added to the pipeline for the designated salesperson.
+
+   To enable the :guilabel:`Leads` feature, navigate to :menuselection:`CRM app --> Configuration
+   --> Settings` and tick the :guilabel:`Leads` checkbox.
+
+Pricing
+=======
+
+Lead mining is an *In-App Purchase* feature, and each generated lead costs one :ref:`credit
+<in_app_purchase/credits>`.
+
+.. important::
+   Generating |CC| costs one additional credit for each contact generated. See here for complete
+   pricing information: `Lead Generation by Odoo IAP
+   <https://iap.odoo.com/iap/in-app-services/167?>`_.
 
 To buy credits, navigate to :menuselection:`CRM --> Configuration --> Settings`. In the
 :guilabel:`Lead Generation` section, under the :guilabel:`Lead Mining` feature, click on
 :guilabel:`Buy Credits`.
 
-.. image:: lead_mining/buy-lead-mining-credits-setting.png
-   :align: center
-   :alt: Buy credits from the lead mining settings.
-
 Credits may also be purchased by navigating to the :menuselection:`Settings --> General Settings`.
```

Comment:
```suggestion
Credits may also be purchased by navigating to the :menuselection:`Settings app --> General Settings`.
```
Since the first thing seen when opening the Settings app *is* the General Settings page, is this additional step in the menuselection necessary? What do you think?

Reply:
Probably not, I will update :+1: